src/parsers/xyz.ts

```diff
@@ -134,7 +174,9 @@ function fromBasis(
     const XYZArray: string[] = [];
     basisClsInstance._elements.array.forEach((item, idx) => {
         // assume that _elements and _coordinates are indexed equivalently
-        const element = s.sprintf("%-3s", item);
+        const atomicLabel = basisClsInstance.atomicLabelsArray[idx];
+        const elementWithLabel = item + atomicLabel;
+        const element = s.sprintf("%-3s", elementWithLabel);
```

Comment:
As we discussed, let's create a task to generalize the labels, pls, wrt to:
- utilizing any string as label
- making the numeric space larger than just 0-9
- allowing to store metadata (perhaps just the description field, like in webapp) to explain what the labels mean